src/mac/activate-apps-cmake.sh

```diff
@@ -28,7 +28,7 @@ ivlibdir=$4
 export CPU
 export NRN_SRC
 NSRC=$NRN_SRC
-export NSRC #needed by nrnversion.sh
+export NSRC # needed by nrnversion.sh
```

Comment:
This has a substantive effect? I really don't understand how it fixes "syntax error near unexpected token `do"

Reply:
that was quite misleading because of `CTLF` as end of line char : https://blog.papercut.com/solved-syntax-error-near-unexpected-token-do/